base/uk.ac.stfc.isis.ibex.configserver/src/uk/ac/stfc/isis/ibex/configserver/editing/EditableConfiguration.java

```diff
@@ -298,7 +329,7 @@ public Group apply(EditableGroup group) {
      * @return The IOCs associated with the configuration
      */
     private Collection<Ioc> getIocs() {
```

Comment:
Looking through this code, I don't see where its used that getSelectedIocs couldn't be instead. Can we remove it?